app/modules/gdbstub.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Module for doing various horrible things 
```

Comment:
Technically, I can't argue with that statement, but something a teensy bit more descriptive might be nice :)